modules/rostests/apitests/ntdll/CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ add_rostests_file(TARGET ntdll_apitest SUBDIR testdata)
 # These are empty files
 add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/ntdll_apitest.exe.local" SUBDIR testdata)
 add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/shell32.dll" SUBDIR testdata)
-add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/test.dll" SUBDIR testdata)
+# add_rostests_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/test.dll" SUBDIR testdata)
```

Comment:
Commit "[NTDLL_APITEST] Try to fix test for NtMaßViewOfSection" --> should read "NtMapViewOfSection"

Reply:
Fixed